types/conf.go

```diff
@@ -202,6 +212,9 @@ func CreateCNIRuntimeConf(args *skel.CmdArgs, k8sArgs *K8sArgs, ifName string, r
 		if len(delegateRc.InfinibandGUID) != 0 {
 			capabilityArgs["infinibandGUID"] = delegateRc.InfinibandGUID
 		}
+		if delegateRc.DeviceID != "" {
```

Comment:
Does this mean we need to add `DeviceID` capability explicitly when creating net-attach-def?

Reply:
Only if the CNI supports deviceIDs not coming from DevicePlugin, e.g: host-device